nuntium/templates/write/missing_contacts.html

```diff
@@ -0,0 +1,32 @@
+{% extends "base_instance.html" %}
+{% load i18n %}
+{% load static %}
+
+{% block promo %}
+{% endblock %}
+
+
+{% block content_inner %}
+
+  <h1>{% trans "Help to find missing contact details" context "Title of page appealing for help finding more contact details" %}</h1>
+
+  <p>
+    {% blocktrans trimmed %}
+      We don't have an email address for any of the people
```

Comment:
Tiny thing, but, curly quote here? `don’t`, rather than `don't` 😄 

(We use curly quotes elsewhere, like `"You can’t currently create new messages using this site."` in `who.html`)